### ISSUE TITLE
TypeError: Cannot read property 'point-radius' of undefined

### ISSUE THREAD

**Issue by orcutt989** [d3/d3-geo-projection#117]
Hey Mike - when I try...
```bash
geo2svg -w 960 -h 960 < ca-albers.json > ca-albers.svg
```
I get...
```bash
TypeError: Cannot read property 'point-radius' of undefined
    at transform (/usr/lib/node_modules/d3-geo-projection/bin/geo2svg:49:26)
    at callbackObject (/usr/lib/node_modules/d3-geo-projection/bin/read.js:33:12)
    at process._tickCallback (internal/process/next_tick.js:109:7)
```

I apologize for raising an issue if I'm just doing something wrong. Thanks for your help.

**Comment by mbostock** [comment]
Ah. There’s a bug with null features. One moment please.

**Comment by mbostock** [comment]
Fix released in 2.1.2.

**Comment by orcutt989** [comment]
Thanks!
